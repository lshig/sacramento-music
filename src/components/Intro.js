import React from 'react';
import Accordion from './Accordion';
import Button from './Button';

export default function Intro() {
  return (
    <section>
      <Button />
      <Accordion />
    </section>
  );
}
