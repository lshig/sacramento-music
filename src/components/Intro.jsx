import React from 'react';
import Accordion from './Accordion';
import Button from './Button';
export default function Intro () {
  return (
    <section className="intro">
      <Button />
      <Accordion />
    </section>
  );
}
