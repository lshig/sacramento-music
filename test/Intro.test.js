import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import Intro from '../src/components/Intro';

describe('Intro', () => {
  describe('basic', () => {
    const wrapper = shallow(<Intro />);

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
