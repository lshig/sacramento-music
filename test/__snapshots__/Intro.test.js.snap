// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro basic renders 1`] = `
<section>
  <Button />
  <Accordian />
</section>
`;
